tests/Aspire.Hosting.Tests/Schema/SchemaTests.cs

```diff
@@ -249,13 +242,13 @@ public async Task ManifestAcceptsUnknownResourceType()
             }
             """;
 
-        var manifestJson = JToken.Parse(manifestTest);
-        var schema = await GetSchemaAsync();
-        Assert.True(manifestJson.IsValid(schema));
+        var manifestJson = JsonNode.Parse(manifestTest);
+        var schema = GetSchema();
+        Assert.True(schema.Evaluate(manifestJson).IsValid);
```

Comment:
Just curious - why not use `AssertValid(..)` everywhere?

Reply:
I guess I was just migrating the code as it is. I can change it though.